server/types/interface.go

```diff
@@ -113,3 +126,7 @@ var typesFromBaseID = map[DoltgresTypeBaseID]DoltgresType{
 	Xid.BaseID():              Xid,
 	XidArray.BaseID():         XidArray,
 }
+
+func GetAllPgTypes() map[string]DoltgresValidType {
```

Comment:
We should add a function comment here, and we may want to return an array instead, just to prevent some other code accidentally modifying the map. Also change this to use `typesFromBaseID`, as per one of the other comments.